cmd/zetae2e/local/local.go

```diff
@@ -407,3 +351,26 @@ func waitKeygenHeight(
 		logger.Info("Last ZetaHeight: %d", response.Height)
 	}
 }
+
+func must[T any](v T, err error) T {
```

Comment:
is `mustSucceed` a better name?

Reply:
Idea with `must` is that it prefix the function name
GetBool -> mustGetBool